### PR TITLE
Render Snake

### PR DESCRIPTION
**Pull request by nickhallph**
- Do it

### CODE REVIEW

File: snake_game/SnakeGame/main.cpp [hunk 1]
```diff
@@ -11,15 +11,17 @@ int main()
 	// Create new map
 	Map *MapInstance = new Map;
 
-	// Set the game intro
+	// Start a new game
+	// do {
+	// 	std::cout << NewGame->StartGame() << std::endl;
+	// } while (NewGame->GetIsGameOver() != true);
```
Comment (nickkhall) [inline snake_game/SnakeGame/main.cpp:17]: Be careful there cowboy, you need that kill switch functionality before you can run this.

Comment (nickhallph) [reply snake_game/SnakeGame/main.cpp:17]: UH DUH IDIOT. I'm just as smart as you are.

Comment (nickkhall) [reply snake_game/SnakeGame/main.cpp:17]: true-true





....good point

**Review by nickkhall — APPROVED** [review]
good stuff, good stuff

**Pull request merged**